python-spec/src/somacore/query/types.py

```diff
@@ -26,7 +26,7 @@ def get_indexer(
         """Something compatible with Pandas' Index.get_indexer method."""
 
 
-IndexFactory = Callable[[npt.NDArray[np.int64]], "IndexLike"]
+IndexFactory = Callable[[npt.NDArray[np.int64], Optional[Any]], "IndexLike"]
```

Comment:
The reason we have this take the NDArray as the only parameter is so that we can use the function `pandas.Index` as an index factory. With this added, `pandas.Index` no longer works, which is why the format check is now failing.